src/middlewares/protected.js

```diff
@@ -0,0 +1,32 @@
+import jwt from 'jsonwebtoken';
+
+class ProtectedRoute {
+  static verifyToken = async (req, res, next) => {
```

Comment:
a better name would be "superadminAuth" since it checks if the user is a super admin rather than if they are verified